cdp_scrapers/instances/seattle.py

```diff
@@ -47,6 +47,10 @@
 if known_persons:
     log.debug(f"loaded static data for {', '.join(known_persons.keys())}")
 
+# we have discovered the city clerk accidentally entered Daniel Strauss
+# instead of the correct Dan Strauss for a few events
+person_aliases = {"Dan Strauss": set(["Daniel Strauss"])}
```

Comment:
Typically, constants are all caps. `PERSON_ALIASES`

Reply:
Shoot, of course.